source/MaterialXGraphEditor/CMakeLists.txt

```diff
@@ -88,7 +94,7 @@ target_link_libraries(
     MaterialXGraphEditor
     PRIVATE
     ${MATERIALX_LIBRARIES}
- 	glfw_minimal)
+ 	glfw)
```

Comment:
This specific line change may need additional clarification, as we're still using the "glfw_minimal" name for the unique copy of GLFW in the MaterialX Graph Editor:

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/source/MaterialXGraphEditor/External/Glfw/CMakeLists.txt#L44

Is this unique name no longer needed, or are there perhaps situations where we should continue referencing the "glfw_minimal" name?

Reply:
I wonder if changing the name to `glfw` is causing the build process to find a different installation of GLFW entirely, not the one that's provided in the MaterialX Graph Editor?

The motivation for the unique name is that CMake doesn't allow two projects with the same name to exist within a build, and NanoGUI already contains a copy of GLFW using its default name:

https://github.com/mitsuba-renderer/nanogui/tree/f5020e2f3e5114d517642e67afbb21cb88cf04c0/ext

Reply:
your hunch was correct - I had a homebrew install of glfw that was confusing things - once I removed that - the `glfw_minimal` target worked. sorry about that.

Reply:
Thanks for verifying, and otherwise this change looks good to me!